guides/database-integrating-with-postgresql.adoc

```diff
@@ -0,0 +1,739 @@
+
+Replace the current `wildfly-maven-plugin` configuration provided by the getting starter guide with the following one in the `pom.xml` file:
+
+[source,xml]
+----
+<plugin>
+    <groupId>org.wildfly.plugins</groupId>
+    <artifactId>wildfly-maven-plugin</artifactId>
+    <version>${version.wildfly.maven.plugin}</version>
+    <configuration>
+        <discover-provisioning-info>
+            <addOns>
+                <addOn>postgresql:default</addOn>
+            </addOns>
+        </discover-provisioning-info>
+    </configuration>
+    <executions>
+        <execution>
+            <goals>
+                <goal>package</goal>
+            </goals>
+        </execution>
+    </executions>
+</plugin>
+----
+In the above configuration, behind scenes the `wildfly-maven-plugin` is using https://docs.wildfly.org/wildfly-glow/[WildFly Glow, window=_blank] to discover automatically the required Galleon Layers for our application. The `discover-provisioning-info` configuration tells the plugin to discover the required layers by inspecting our application code. By using `postgresql:default` addon, we are specifying we want to use a PostgreSQL database, and we want to configure it as the default datasource for the server.
+
+=== persistence.xml
+This file is used to configure the Jakarta Persistence Api (JPA) unit and the schema generation strategy. In this guide, we are using the `drop-and-create` strategy to drop the existing schema and create a new one every time the application starts. For a production environment, you should use a more appropriate strategy to avoid data loss.
+
+Create the following `persistence.xml` file in the `src/main/resources/META-INF` directory:
+
+[source,xml]
+----
+<?xml version="1.0" encoding="UTF-8" ?>
+<persistence xmlns="https://jakarta.ee/xml/ns/persistence"
+             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="https://jakarta.ee/xml/ns/persistence https://jakarta.ee/xml/ns/persistence/persistence_3_0.xsd"
+             version="3.0">
+    <persistence-unit name="bookstore-PU">
+        <properties>
+            <property name="jakarta.persistence.schema-generation.database.action" value="drop-and-create"/>
+        </properties>
+    </persistence-unit>
+</persistence>
+----
+NOTE: We don't need to specify the name of the Datasource by using `<jta-data-source>`. In absence of this property, JPA will use the default datasource configured in the server.
+
+=== Configure RESTful Web Services (JAX-RS) application
+
+The `BookStoreApplication` class acts as a configuration class for the JAX-RS application. It essentially tells the JAX-RS runtime that this is a JAX-RS application and provides the base path for the application's RESTful web services.
```

Comment:
s/tells the JAX-RS runtime/tells the WildFly runtime/g

The fact that it's the jax-rs subsystem that detects this is an internal detail of the WildFly runtime.

Reply:
> Jakarta REST.
> We should not use the term JAX-RS beyond the very first "Jakarta RESTful Web Services (JAX-RS)" above.

Not sure if I am following you here. Should not we use JAX-RS to avoid writing the full name? I understand that in the title of a section makes more sense to use the full name "Jakarta RESTful Web Services" (with or without the JAX-RS since it has been already mentioned before), and then in the paragraph use JAX-RS to avoid using the full name.

Reply:
@yersan JAX-RS is a Java EE trademark that was not transferred by Oracle to the Eclipse foundation when Jakarta was created. We should avoid using it as a shorthand -- use Jakarta REST.  Similar things apply to all the old Java EE shorthands, JAX-WS etc. We are not required or willing to break our software to deal with this change (e.g. renaming subsystems) but for purely documentation things like this we should avoid the old shorthands.

TBH, we shouldn't use JAX-RS even once. This shift from Java EE to Jakarta EE is many years old now; we don't need to remind people who are actually using these technologies.

I thought that you were trying to remind people of the old term by using it, but I think now you were more just trying to use a short form. Don't use JAX-RS as a short form.

Reply:
@bstansberry ok, I've have updated it accordingly, take another look when you have a chance, thanks!